src/pool/srv_cli.c

```diff
@@ -1224,3 +1224,90 @@ dsc_pool_svc_upgrade(uuid_t pool_uuid, d_rank_list_t *ranks, uint64_t deadline)
 	D_DEBUG(DB_MGMT, DF_UUID ": Upgrading pool prop\n", DP_UUID(pool_uuid));
 	return dsc_pool_svc_call(pool_uuid, ranks, &pool_upgrade_cbs, NULL /* arg */, deadline);
 }
+
+struct pool_rebuild_stop_arg {
+	uint32_t rstpa_force;
+};
+
+static int
+pool_rebuild_stop_init(uuid_t pool_uuid, crt_rpc_t *rpc, void *varg)
+{
+	struct pool_rebuild_stop_arg *arg = varg;
+	struct pool_rebuild_stop_in  *in;
+
+	in              = crt_req_get(rpc);
+	in->rstpi_force = arg->rstpa_force;
+	return 0;
+}
+
+static int
+pool_rebuild_stop_consume(uuid_t pool_uuid, crt_rpc_t *rpc, void *varg)
+{
+	struct pool_rebuild_stop_out *out = crt_reply_get(rpc);
+	int                           rc  = out->rstpo_op.po_rc;
+
+	if (rc != 0)
+		DL_ERROR(rc, DF_UUID ": failed to stop rebuild", DP_UUID(pool_uuid));
+	return rc;
+}
+
+static struct dsc_pool_svc_call_cbs pool_rebuild_stop_cbs = {.pscc_op   = POOL_REBUILD_STOP,
+							     .pscc_init = pool_rebuild_stop_init,
+							     .pscc_consume =
+								 pool_rebuild_stop_consume,
+							     .pscc_fini = NULL};
```

Comment:
[Nit] Might benefit from turning off clang-format? But I'd not refresh the PR only for addressing this.